apps/coa/sections/summary-data-sections.js

```diff
@@ -82,6 +82,20 @@ module.exports = {
               return field.parsed;
             }).join('\n')).join('\n \n');
         }
+      },
+      {
+        step: '/postal-address',
+        field: 'postal-address-details',
+        parse: (list, req) => {
+          if (!req.sessionModel.get('steps').includes('/postal-address')) {
+            return null;
+          }
```

Comment:
Instead I would suggest check agains the `which-details-updating` if that contain the value `postal-address`

Please keep in ming that `which-details-updating` could return String or Array

Reply:
Withdrawn. For future discussion in a wider context within a team